src/poliastro/threebody/restricted.py

```diff
@@ -0,0 +1,173 @@
+# coding: utf-8
+
+import numpy as np
+from scipy.optimize import root
+
+
+def lagrange_points(r12, m1, m2):
+    """Computes the Lagrangian points using the formulation
+    in Problem 8-5 of Battin, Richard H. 'An introduction to 
+    the mathematics and methods of astrodynamics'. AIAA, 1999.
+
+    Parameters
+    ----------
+    r12 : float
+        Collinear distance
+    m1 : float
+        Mass of the main body
+    m2 : float
+        Mass of the secondary body
+
+    Returns
+    -------
+    array(float)
+        Distance of the Lagrangian points to the center,
+        projected on the collinear line
+    """
+
+    rho = r12
+
+    def eq_L1(rho2):
+        aux = 3 * rho**2 * rho2 - 3 * rho * rho2**2 + rho2**3
+        aux *= rho2**2
+        aux /= rho**3 - rho2**3
+        aux /= (rho - rho2)**2
+        aux -= m2 / m1
+        return aux
+
+    def eq_L2(rho2):
+        aux = 3 * rho**2 * rho2 + 3 * rho * rho2**2 + rho2**3
+        aux *= rho2**2
+        aux /= rho**3 - rho2**3
+        aux /= (rho + rho2)**2
+        aux -= m2 / m1
+        return aux
+
+    def eq_L3(rho1):
+        aux = 3 * rho**2 * rho1 + 3 * rho * rho1**2 + rho1**3
+        aux *= rho1**2
+        aux /= rho**3 - rho1**3
+        aux /= (rho + rho1)**2
+        aux -= m1 / m2
+        return aux
+
+    l = np.zeros((5,))
+
+    # L1
+    rho2 = root(eq_L1, 0).x
+    rho1 = rho - rho2
+    l[0] = rho1
+
+    # L2
+    rho2 = root(eq_L2, 0).x
+    rho1 = rho + rho2
+    l[1] = rho1
+
+    # L3
+    rho1 = root(eq_L3, -1000).x
+    # rho2 = rho + rho1
+    l[2] = - rho1
+
+    # TODO: add checks to the results given by `root`
+    # TODO: reassure that the initial values given to `root` are good for all cases
+
+    l[3] = l[4] = 0.5 * r12
```

Comment:
Don't L4 and L5 distances depend on the mass ratio as well?
![screenshot from 2018-08-17 19-37-14](https://user-images.githubusercontent.com/316517/44280412-fcfa1300-a254-11e8-98cd-80ab48c24cb3.png)

(Again from Curtis)

Reply:
![imagen](https://user-images.githubusercontent.com/28366329/44285711-b9a8a000-a266-11e8-9a02-b5abaf985d1a.png)

This is from Battin, so... May be Curtis has more updated information.
Also, I'd like to give it a look before jumping into conclusions, but maybe Curtis' equation uses a different frame of reference? With the origin in the barycenter? I'll give it a look

Reply:
Oh, interesting! Whatever the result is, please leave a comment explaining it so nobody makes the mistake of changing it in the future.

Reply:
Yeah, Curtis uses a frame of reference (*comoving coordinate system*) with the origin in the barycenter (Fig 2.33), so the formulation seems to be consistent.